theano/sandbox/jax_linker.py

```diff
@@ -76,7 +76,7 @@ def create_jax_thunks(self, compute_map, storage_map):
 
             # JIT-compile the functions
             if len(node.outputs) > 1:
-                assert len(jax_funcs) == len(node.ouptputs)
+                assert len(jaxed_fgraph_outputs) == len(node.outputs)
```

Comment:
This had a typo before, probably need a test to cover this case.

Reply:
I don't believe the `jax_funcs` part is incorrect; there should be a JAXable function for each node output, which is what that assert is stating (aside from the typo).

Reply:
Hm, there was only one for multi-output func graphs.